test/jdk/java/lang/management/MemoryMXBean/LowMemoryTest2.sh

```diff
@@ -71,7 +71,7 @@ go -noclassgc -XX:MaxMetaspaceSize=32m -XX:+UseParallelGC LowMemoryTest2
 #  can sit atop of uncommitted address space, denying new loaders address space. In the old Metaspace,
 #  we would have committed the space right away and therefore the MemoryMXBean "committed" trigger
 #  would have fired. In the new Metaspace, we don't commit, so the MemoryMXBean does not fire.
-go -noclassgc -XX:MaxMetaspaceSize=16m -XX:CompressedClassSpaceSize=4m LowMemoryTest2
+go -noclassgc -XX:MaxMetaspaceSize=4m LowMemoryTest2
```

Comment:
Why did you reduce the MaxMetaspaceSize? (Should probably be okay post JEP-387, but how does it relate to your change?)

Reply:
Maybe I misunderstand the purpose of LowMemoryTest2?

I thought the original test case 
```bash
go -noclassgc -XX:MaxMetaspaceSize=16m -XX:CompressedClassSpaceSize=4m LowMemoryTest2
```
is used to check if low memory ratio(20% remaining) of compressed class pool could be detected instead of throwing Compressed Class Space OOM. Since we don't have compressed class pool anymore, I thought metaspace should be reduced to adapt original purpose of that test case.